bin/test_sugardough_docker.sh

```diff
@@ -1,8 +1,26 @@
 #!/bin/bash
-set -e
+#
+# Runs unit_tests
+#
+set -ex
+# Create a temporary virtualenv to install docker-compose
+ENVDIR=`mktemp -d`
+virtualenv $ENVDIR
+. $ENVDIR/bin/activate
+pip install cookiecutter docker-compose
 
 TDIR=`mktemp -d`
-virtualenv $TDIR
-. $TDIR/bin/activate
-pip install tox cookiecutter
-TOX_ENV=dockertests ./bin/test_sugardough.sh
+cp cookiecutter.json $TDIR/
+cd $TDIR
+cookiecutter --no-input $OLDPWD
+cd sugardough
+
+DC_CMD="docker-compose --project-name jenkins${JOB_NAME}${BUILD_NUMBER}"
+
+$DC_CMD run -T web flake8 sugardough
+
+# Run Tests
+$DC_CMD run -T web ./manage.py test
+
+# Delete virtualenv
+rm -rf $TDIR
```

Comment:
Since there are now 2 temp dirs, shouldn't we also `rm -rf $ENVDIR` and update the comment?